runtime/browser/code-notes.md

```diff
@@ -0,0 +1,55 @@
+## Notes on rendering pipeline
+
+```js
+DomParticle::(inner)
+  setViews() => _renderViews()
+  _renderViews() => _renderToSlot()
+  _renderToSlot() => slot<let>.render() 
+  this._pec._apiPort.RenderSlot() => isolate => this.slotManager.renderSlot
```

Comment:
can you, please, rename slotManager to slotComposer here (and below in this file)?